utils/kubelet.go

```diff
@@ -0,0 +1,95 @@
+package utils
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/containernetworking/cni/pkg/types"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func PodKey(pod *corev1.Pod) string {
```

Comment:
I'm reluctant to make `utils` an actual grab-bag of random things. I would rather have a separate k8sutils